tests/common/plugins/pdu_controller/__init__.py

```diff
@@ -30,15 +30,28 @@ def get_pdu_visible_vars(inventories, pdu_hostnames):
     return pdu_hosts_vars
 
 
+def get_sup_node_or_first_node(duthosts):
```

Comment:
can we define this function in common file in `utilities.py`?

Reply:
will do, thanks